client/realtor.lua

```diff
@@ -35,6 +51,63 @@ local function stopPreview()
     end
 end
 
+local function addGaragePoint()
+    isAddingGarage = not isAddingGarage
+    garageCoords = nil
+
+    showText()
+    spawnCar()
+
+    while isAddingGarage do
+        Wait(0)
+
+        local hit, _, endCoords, _, _ = lib.raycast.fromCamera(511, 4, 25.0)
```

Comment:
No need for the last two underscores, you can remove those completely. You only need to keep them if they're in between variables.